packages/nextjs/src/pages/ClerkProvider.tsx

```diff
@@ -80,6 +78,7 @@ export function ClerkProvider({ children, ...rest }: NextClerkProviderProps): JS
       initialState={authServerSideProps?.__clerk_ssr_state || __clerk_ssr_state}
       {...restProps}
     >
+      <h2>pages/clerkprovider</h2>
```

Comment:
✂️

Reply:
Need it for testing, will remove right after :)